ee/candi/cloud-providers/openstack/openapi/cluster_configuration.yaml

```
@@ -155,7 +161,7 @@ apiVersions:
                 items:
                   type: string
                 uniqueItems: true
-              additionalTags:
+              additionalTags: &instanceClassAdditionalTags
```

```suggestion
              additionalTags: &instanceClassAdditionalTags
```

As I know, we agreed not to use YAML anchors.